terraform/backends/dpc-test-gf.s3.tfbackend

```diff
@@ -0,0 +1,2 @@
+bucket         = "dpc-test-tfstate-20250410145524530000000001"
+use_lockfile   = true
```

Comment:
There are extra spaces before the equal signs in these files. Run `terraform fmt backends/*` in the terraform directory.